modules/transactions.js

```diff
@@ -187,6 +187,33 @@ __private.getById = function (id, cb) {
 	});
 };
 
+__private.getVotesById = function (transaction, cb) {
+	library.db.query(sql.getVotesById, {id: transaction.id}).then(function (rows) {
+		if (!rows.length) {
+			return setImmediate(cb, 'Transaction not found: ' + id);
+		}
+
+		var votes = rows[0].votes.split(',');
+		var added = [];
+		var deleted = [];
+
+		for (var i = 0; i < votes.length; i++) {
+			if (votes[i].substring(0, 1) == "+") {
```

Comment:
Sure: [gist](https://gist.github.com/vidul-nikolaev-petrov/a6f615373bbb0fd3fd985418926606fa) and [online](https://goo.gl/VTDzzO
) example. Tested on localhost on _v4.6.1_ and _v6.9.0_ -- similar results.